.github/RELEASE_SCHEDULE.md

```diff
@@ -1,16 +1,29 @@
 # DNN Platform Release Schedule
-To ensure adequate time for release planning by the community, partners, and vendors a specific release process will be followed for all releases.  
+To ensure adequate time for release planning by the community, partners, and vendors, a specific release process will be followed for all releases.  
 
 ## Release Candidates
-For a period of one week (Revision), two weeks (Minor) or three-weeks (Major) before any release, a Release Candidate (RC) version will be made available to the public.  At present these release candidates will be for testing only. After version 10.x efforts will be made to support upgrading from RC to Production releases.
+Releases candidates (RC) are designed to give the community time to adjust their existing environments for any breaking changes and identify any unintended changes. Strong community participation during the RC process will result in more stable releases. 
 
-The goal of these release candidates is to give the community time to adjust their existing environments for any breaking changes, as well as to identify any issues with the changes.  If necessary, changes will be incorporated an additional RC release could be made if significant problems are identified.  If a revised Release Candidate is necessary the Production Release schedule may be impacted.  The exact impact will vary on a case-by-case basis depending on the nature of the issue(s) identified during RC review, however, will be clearly communicated as part off the updated RC release notes.
+### Major Releases (Example 10.0.0)
```

Comment:
Not sure I agree on this one, this still requires thinking

Reply:
@mitchelsellers this was just a way to keep it structured and so that we don't outdated it by giving random examples.  This is consistent with our Milestones too.